<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CtrlMoney</class>
 <widget class="QMainWindow" name="CtrlMoney">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Менеджер финансов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="framePanel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="spinBoxYear">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>20</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="minimum">
          <number>2000</number>
         </property>
         <property name="maximum">
          <number>2050</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxMonth">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="maxVisibleItems">
          <number>12</number>
         </property>
         <item>
          <property name="text">
           <string>Январь</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Февраль</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Март</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Апрель</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Май</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Июнь</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Июль</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Август</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Сентябрь</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Октябрь</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ноябрь</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Декабрь</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonAddExpense">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonAddIncome">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttomAbout">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabData">
       <attribute name="title">
        <string>Журнал</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
        <item>
         <widget class="QFrame" name="frameSummary">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>70</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <widget class="QLabel" name="labelIncome">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>151</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>ИТОГО Расход/Доход:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLineEdit" name="editExpense">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>10</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0.00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="editIncome">
           <property name="geometry">
            <rect>
             <x>300</x>
             <y>10</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0.00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="editBalance">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>40</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0.00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="labelBalance">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>151</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Баланс:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabChart">
       <attribute name="title">
        <string>Диаграмма</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="vboxChart"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
